prql-compiler/src/ast/pl/ident.rs

```diff
@@ -40,6 +42,12 @@ impl Ident {
         }
     }
 
+    pub fn prepend(self, part: String) -> Ident {
+        let mut parts = vec![part];
+        parts.extend(self.into_iter());
+        Ident::from_path(parts)
```

Comment:
nit if you think concise is better, discard if not
```suggestion
    Ident::from_path(std::iter::once(part).chain(self.into_iter()).collect())
```

Reply:
Nah, i think using vec is more readable than iterators.